test/objectwrap.cc

```diff
@@ -115,64 +120,120 @@ class Test : public Napi::ObjectWrap<Test> {
     Napi::Symbol kTestMethodTInternal     = Napi::Symbol::New(env, "kTestMethodTInternal");
     Napi::Symbol kTestVoidMethodTInternal = Napi::Symbol::New(env, "kTestVoidMethodTInternal");
 
-    exports.Set("Test", DefineClass(env, "Test", {
```

Comment:
This one and many lines below look like pure whitespace changes. Are they necessary?

Reply:
There are two new elements at the bottom of the block. git-clang-format will check with blocks as minimum units.